pkg/controller/common/helpers.go

```diff
@@ -146,13 +125,47 @@ func WriteTerminationError(err error) {
 	glog.Fatal(msg)
 }
 
-// convertIgnition3to2 takes an igntion v3 config and returns a v2 config
-func convertIgnition3to2(ignconfig ign3types.Config) (ign2types.Config, error) {
-	converted2, err := ignconverter.Translate3to2(ignconfig)
+// ConvertIgnition2to3 takes an ignition spec v2 config and returns a v3 config
+func ConvertIgnition2to3(ign2config ign2types.Config) (ign3types.Config, error) {
+	// only support writing to root file system
+	fsMap := map[string]string{
+		"root": "/",
+	}
+
+	// Workaround to get v2.3 as input for converter
+	ign2_3config := ign2_3.Translate(ign2config)
+	ign3_0config, err := ignconverter.Translate(ign2_3config, fsMap)
+	if err != nil {
+		return ign3types.Config{}, errors.Errorf("unable to convert Ignition spec v2 config to v3: %v", err)
+	}
+	// Workaround to get a v3.1 config
+	converted3 := translate3.Translate(ign3_0config)
+
+	glog.V(4).Infof("Successfully translated Ignition spec v2 config to Ignition spec v3 config: %v", converted3)
+	return converted3, nil
+}
+
+// ConvertIgnition3to2 takes an ignition spec v3 config and returns a v2 config
+func ConvertIgnition3to2(ign3config ign3types.Config) (ign2types.Config, error) {
+	// Bad hack to convert spec 3.1 config to spec 3.0
+	// ign-convert doesn't currently support spec v3.1
+	// TODO(lorbus)
```

Comment:
So, I'm tempted to say we should just be on 3.0 for now to not add confusion, and wait until we're sure what we want for https://github.com/coreos/ign-converter/pull/6. Then we can add a hack to convert from 3.0 to 3.1 instead of the 3.1  to 3.0 here. I could be missing something, is there functionality you need from 3.1 or just trying to update to latest stable?